filer/static/filer/js/base.js

```diff
@@ -48,7 +48,9 @@ Cl.mediator = new Mediator();
             if (event.type === 'focus') {
                 container.addClass('is-focused');
             } else {
-                container.removeClass('is-focused');
+                if (event.relatedTarget == undefined) {
```

Comment:
never `==` compare to undefined
should be something like `if (event.relatedTarget && event.relatedTarget.is(dropdownTrigger) {`